### ISSUE TITLE
Local modules are not represented correctly in the graph

### ISSUE THREAD

**Issue by jorgecoca** [vanniktech/gradle-dependency-graph-generator-plugin#39]
On a multi-module environment, a local module is being "duplicated" in the graph representation:

![dependency-graph](https://user-images.githubusercontent.com/3237451/38895959-0f802572-4257-11e8-8bbd-0df9b5487bda.png)

As you can see in the image, "network" is being duplicated as a node, when in reality it is just a "submodule" of app in this case.

**Comment by jorgecoca** [comment]
In that same example, you can also see the "logger" module presenting the same behavior.

<img width="213" alt="screen shot 2018-04-17 at 5 30 34 pm" src="https://user-images.githubusercontent.com/3237451/38902189-1c690ea8-4265-11e8-8c92-07d3b45c03a6.png">

**Comment by vanniktech** [comment]
Yes. Try the 0.4.0-SNAPSHOT version please. That should be fixed there.

**Comment by jorgecoca** [comment]
Run it with Snapshot, and problem is fixed! Thanks!

![dependency-graph](https://user-images.githubusercontent.com/3237451/38999888-6530d586-43b8-11e8-8281-1eb637641ea2.png)
